src/main/kotlin/com/example/fhirvalidator/configuration/ApplicationConfiguration.kt

```diff
@@ -27,8 +26,12 @@ class ApplicationConfiguration {
     }
 
     @Bean
-    fun fhirServerR4Registration(fhirContext: FhirContext, npmPackages: List<NpmPackage>): ServletRegistrationBean<*> {
-        val registration: ServletRegistrationBean<*> = ServletRegistrationBean<Servlet>(FHIRRestfulServer(fhirContext, applicationContext, npmPackages), "/R4/*")
+    fun fhirServerR4Registration(
+        fhirRestfulServer: FHIRRestfulServer
+    ): ServletRegistrationBean<Servlet> {
+        val registration: ServletRegistrationBean<Servlet> = ServletRegistrationBean(
+            fhirRestfulServer, "/R4/*"
+        )
         val params: MutableMap<String, String> = HashMap()
         params["FhirVersion"] = "R4"
```

Comment:
what do these params do? (can we remove them)

Reply:
Matt maybe be able to reconfigure. I couldn't get spring beans to work correctly.

Reply:
No we don't